x-pack/dev-tools/api_debug/apis/telemetry/index.js

```diff
@@ -8,6 +8,6 @@
 export const name = 'telemetry';
 export const description = 'Get the clusters stats from the Kibana server';
 export const method = 'POST';
-export const path = '/api/telemetry/v2/clusters/_stats';
+export const path = '/internal/telemetry/clusters/_stats';
```

Comment:
I noticed this script already supports the internal header but doesn't support the version header.

We might need to solve it in a follow-up PR.